qiskit_experiments/framework/experiment_data.py

```diff
@@ -82,3 +66,13 @@ def completion_times(self) -> Dict[str, datetime]:
                 job_times[job_id] = job.time_per_step().get("COMPLETED")
 
         return job_times
+
+    def __repr__(self):
```

Comment:
Should this be in `DbExperimentDataV1`?

Reply:
I added one to each. But since they have quite different init args i thought it best to do different repr for each one.